src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/ProcessDependencyResult.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.DotNet.UpgradeAssistant.Analysis;
+using Microsoft.DotNet.UpgradeAssistant.Dependencies;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Packages
+{
+    public class ProcessDependencyResult : IProcessResult
+    {
+        public IList<Run> RunProcessResult(string toolName, Dictionary<string, IDependencyAnalysisState> analysisResults)
+        {
+            if (analysisResults is null)
+            {
+                throw new ArgumentNullException(nameof(analysisResults));
+            }
+
+            var runs = new List<Run>();
```

Comment:
Where does the Run type come from? Is that a sarif thing?

Reply:
yes Run is a sarif type